spec/Generics.tex

```diff
@@ -858,23 +843,11 @@ \subsection{User-Defined Constructors}
 this is g2
 {c1 = 222, v1 = 222.2, x1 = 22.0, c5 = 3.3, v5 = 333, x5 = 3333}
 \end{chapeloutput}
-The arguments \chpl{t1}, \chpl{p1}, \chpl{c1}, and \chpl{v1} are
-required in all constructors for \chpl{MyGenericClass}. They can appear
-in any order. Both \chpl{MyGenericClass} constructors initialize the
-corresponding fields implicitly because these fields do not have initialization
-expressions. The second constructor also initializes implicitly
-the fields \chpl{t5} and \chpl{p5} because they are a type field
-and a parameter field. It does not initialize the fields \chpl{c5}
-and \chpl{v5} because they have initialization expressions, or
-the fields \chpl{x1} and \chpl{x5} because they are not generic fields
-(even though they are of generic types).
+The initializers are required to initialize fields \chpl{t1}, \chpl{p1},
+\chpl{c1}, and \chpl{v1}. Otherwise, field initializations may be omitted
+according to previously-described initializer semantics.
```

Comment:
I really dislike this c1-p5 example and wish we would just throw it out. Your call though.